crates/db_schema_file/src/schema.rs

```diff
@@ -58,6 +58,16 @@ pub mod sql_types {
   pub struct VoteShowEnum;
 }
 
+diesel::table! {
+    admin_add (id) {
+        id -> Int4,
+        mod_person_id -> Int4,
```

Comment:
The only other thing I can think of, is to possibly rename these `mod_person_id` columns to `admin_person_id` . Doesn't matter to me tho, you can merge as you see fit.